frontend/packages/signless-transactions/src/components/signless-active/signless-active.tsx

```diff
@@ -0,0 +1,22 @@
+import { useAccount } from '@gear-js/react-hooks';
+import { useSignlessTransactions } from '../../context';
+import { ReactComponent as Signless } from '../../assets/icons/signless.svg';
+import styles from './signless-active.module.css';
+
+function SignlessActive() {
+  const { account } = useAccount();
+  const { session, isSessionReady } = useSignlessTransactions();
+
+  return (
+    account &&
```

Comment:
Let's use ternary operator and return `null` for conditional rendering in a standalone components